ext/pgsql/pgsql.c

```diff
@@ -4246,6 +4280,7 @@ PHP_PGSQL_API int php_pgsql_meta_data(PGconn *pg_link, const char *table_name, z
 	zval elem;
 
 	if (!*table_name) {
+	// CHeck if can be TODO
```

Comment:
The error condition itself is clearly promotable, but I haven't checked all callers if they deal with the exception sensibly...

Reply:
I forgot to change this to an assertion, as the only two places where it is called are ``PHP_FUNCTION(pg_meta_data)`` and ``php_pgsql_convert()`` and I made sure all usages of that check that the table name isn't empty.